src/qudi/util/uic.py

```diff
@@ -67,7 +67,7 @@ def loadUi(file_path, base_widget):
             os.remove(file_path)
             raise
     try:
-        result = subprocess.run(f'pyside2-uic "{file_path}"',
+        result = subprocess.run(["pyside2-uic", f"{file_path}"],
```

Comment:
Your current command line input would look like:
```bash
> pyside2-uic /path/to/file
```
Would it not work better if you included the quotation marks in the command in case of whitespaces in `file_path`?
```bash
> pyside2-uic "/path/to a different/file"
```

Reply:
I did not consider this. I tried
`        result = subprocess.run(f'pyside2-uic', f'"{file_path}"')
`
which did revert the situation to the original error for whatever reason,
so I tried something else which seems to work on both platforms.

Reply:
Nevermind. I misunderstood the difference between handing words as a list and giving a full string.
Is there a difference between `['pyside2-uic', file_path]` and `['pyside2-uic', f'"{file_path}"']` on Linux and Windows? I mean including quotation marks in the list?

Reply:
> Nevermind. I misunderstood the difference between handing words as a list and giving a full string. Is there a difference between `['pyside2-uic', file_path]` and `['pyside2-uic', f'"{file_path}"']` on Linux and Windows? I mean including quotation marks in the list?

Found my own answer [here](https://docs.python.domainunion.de/3/library/subprocess.html#frequently-used-arguments)